gslib/commands/mb.py

```diff
@@ -239,6 +253,12 @@ def RunCommand(self):
           storage_class = NormalizeStorageClass(a)
         elif o == '--retention':
           seconds = RetentionInSeconds(a)
+        elif o == '--rpo':
+          rpo = a.strip().upper()
```

Comment:
Would this cause an error if "a" is None?

Reply:
No, the parser will error out that the value is missing before it even reaches here.